### ISSUE TITLE
True italic font support

### ISSUE THREAD

**Issue by mklein994** [termux/termux-app#711]
Currently, italic fonts are achieved by making the current font oblique. It would be great if it would use the italic font instead.

Here's how I envision it could work: Similar to using a custom font with `~/.termux/font.ttf`, put an italic font in a file called `~/.termux/font-italic.ttf`. Then run this:
```bash console
$ termux-reload-settings
$ echo -e '\e[3m light fixture \e[0m' # characters that look different in italics
```
and it would print this:
_` light fixture `_

---
#### Regular:
![screenshot_20180610-172126](https://user-images.githubusercontent.com/3069239/41207046-1009ad7c-6cd4-11e8-9139-3d61c0c0c5e7.png)

#### Italic:
![screenshot_20180610-173553](https://user-images.githubusercontent.com/3069239/41207080-03d8f7f0-6cd5-11e8-9048-5638e45fd6a3.png)

**Comment by LittleHouse75** [comment]
This would be very cool -- hoping to use the font Operator Mono (or one of its free clones) with emacs in Termux.

**Comment by zorro** [comment]
Any updates on this?

**Comment by doas-ice** [comment]
It would be really cool to have this feature. I really love Victor-mono.

**Comment by j0ng4b** [comment]
Well, I successfully build a version of termux thats support italics here some screenshots:

With an italic font:
![Italic](https://user-images.githubusercontent.com/60396154/172279111-99acfc00-0285-448c-8ab7-ce3bc1a29af5.png)


Without an italic font:
![Screenshot_20220606-225524-874](https://user-images.githubusercontent.com/60396154/172279347-a9bf75b3-1aa4-44d0-bb59-a32c6c87dd30.png)

Without italic the text is rendered as a monospaced text. Should I make a pull request? I not already not made one because what I made on code seems a bit of mess and had some points I can't understand.

**Comment by j0ng4b** [comment]
Would be possible to also add true support to bold fonts.

**Comment by zorro** [comment]
> Well, I successfully build a version of termux thats support italics here some screenshots:
> 
> With an italic font: ![Italic](https://user-images.githubusercontent.com/60396154/172279111-99acfc00-0285-448c-8ab7-ce3bc1a29af5.png)
> 
> Without an italic font: ![Screenshot_20220606-225524-874](https://user-images.githubusercontent.com/60396154/172279347-a9bf75b3-1aa4-44d0-bb59-a32c6c87dd30.png)
> 
> Without italic the text is rendered as a monospaced text. Should I make a pull request? I not already not made one because what I made on code seems a bit of mess and had some points I can't understand.

I think it will be better to fallback to the regular font rather than the monospace font

**Comment by j0ng4b** [comment]
> I think it will be better to fallback to the regular font rather than the monospace font

It's true, I forgot to made this, the monospace was to test if everything works as expected, a better fallback is to use the current behaviour of italics alongside the regular font

**Comment by rendiix** [comment]
amazing, waiting for this PR to be merged

![Screenshot_20220621-080109669 (1)](https://user-images.githubusercontent.com/16639978/174695110-075b9d0e-7c35-42ea-8af4-5f1b8fad2f35.jpg)

**Comment by Sodastream11** [comment]
Made another pr similar to the first one since the other was closed because of conflicts

**Comment by aquatix** [comment]
But that newer PR was closed two weeks ago? I'm a bit confused about the status now :) Having real italics support would be great! Thanks for working on this.

**Comment by 11soda11** [comment]
I reimplemented this and opened another pr. I also just found out that the reason why the previous prs werent merged/closed was because of a rewrite currently going on by the maintainer, which is why this last one probably wont be merged for a while too. I will leave it up anyway and refactor it once the rewrite is done but that will take an indefinite amount of time.
